packages/smooth_app/lib/pages/product_page.dart

```diff
@@ -106,89 +98,48 @@ class ProductPage extends StatelessWidget {
-                          ),
-                        )
-                      ],
-                    ),
+                  _getDivider(
+                    Text(appLocalizations.nutrition, style: dividerTextStyle),
                   ),
-                  AttributeExpandable(nutriscoreAttribute),
                   NutritionLevelsExpandable(
-                      nutrientLevels: product.nutrientLevels,
-                      nutriments: product.nutriments),
-                  Padding(
-                    padding: const EdgeInsets.only(
-                        top: 14.0, right: 16.0, left: 16.0, bottom: 8.0),
-                    child: Row(
-                      children: <Widget>[
-                        Flexible(
-                          child: Text(
-                            AppLocalizations.of(context).ingredients,
-                            style: Theme.of(context).textTheme.headline2,
-                          ),
-                        )
-                      ],
-                    ),
+                    product: product,
+                    iconWidth: iconWidth,
+                    attributeTags: const <String>[
+                      UserPreferencesModel.ATTRIBUTE_VEGAN,
+                      UserPreferencesModel.ATTRIBUTE_VEGETARIAN,
+                      UserPreferencesModel.ATTRIBUTE_PALM_OIL_FREE,
+                    ],
+                    title: 'Nutrition levels',
                   ),
-                  ProductProcessingExpandable(
-                    additives: product.additives,
-                    novaGroup: product.nutriments.novaGroup,
+                  NutritionLevelsExpandable(
+                    product: product,
+                    iconWidth: iconWidth,
+                    attributeTags: const <String>[
+                      UserPreferencesModel.ATTRIBUTE_NUTRISCORE,
+                      UserPreferencesModel.ATTRIBUTE_LOW_SALT,
+                      UserPreferencesModel.ATTRIBUTE_LOW_SUGARS,
+                      UserPreferencesModel.ATTRIBUTE_LOW_FAT,
+                      UserPreferencesModel.ATTRIBUTE_LOW_SATURATED_FAT,
+                    ],
+                    title: 'Nutrition levels',
                   ),
-                  Padding(
-                    padding: const EdgeInsets.only(
-                        top: 14.0, right: 16.0, left: 16.0, bottom: 8.0),
-                    child: Row(
-                      children: <Widget>[
-                        Flexible(
-                          child: Text(
-                            AppLocalizations.of(context).ecology,
-                            style: Theme.of(context).textTheme.headline2,
-                          ),
-                        )
-                      ],
-                    ),
+                  _getDivider(
+                    Text(appLocalizations.ingredients, style: dividerTextStyle),
+                  ),
+                  ProductProcessingExpandable(product, iconWidth),
```

Comment:
I think we could treat the product processing attribute groups just like the other attributes, and get all the information needed from the server (e.g. I could put the list of additives in the description of the attribute)

Reply:
> I could put the list of additives in the description of the attribute

I agree, that would make things more standard.
But (cf. https://fr.openfoodfacts.org/produit/5411188115472/soja-original-saveur-douce-alpro):
* the full translated version of the additive should be written (e.g. "E170i - Carbonate de calcium")
* there's an additional parameter "Risque élevé de sur-exposition" that should be included too